provisioning/miniprov/hare_mp/main.py

```diff
@@ -243,11 +249,47 @@ def is_cluster_running() -> bool:
     return os.system('hctl status >/dev/null') == 0
 
 
+def nr_services() -> int:
+    cmd = ['hctl', 'status', '--json']
+    cluster_info = json.loads(execute(cmd))
+    # Don't include hax, count it just once later.
+    services = ['confd', 'ioservice', 's3server']
```

Comment:
```suggestion
    services = {'confd', 'ioservice', 's3server'}
```

Reply:
You mean we don't need list literals but just literals? is it for coding conventions or is there any semantical difference between the 2?

Reply:
done.

Reply:
In your version it is a list, in my case it is a set.